debug/src/debug.js

```diff
@@ -318,12 +318,9 @@ export function initDebug() {
 		// Putting this check in `options.diffed` ensures that
 		// `vnode._children` is set and that we only validate the children
 		// that were actually rendered.
-		if (vnode._children) {
+		if (typeof vnode.type === 'string' && vnode._children) {
```

Comment:
Wondering if we want to only do these checks on DOM nodes or also components.

Reply:
Ah good point - I was forgetting that `._children` is the materialized children, not `props.children` as passed to a component. Will remove.